esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -1663,36 +1659,24 @@ void Cws_machineEx::setProcessInfo(IEspContext& context, CMachineInfoThreadParam
     //set DisplayType
     if (process.getType() && *process.getType())
     {
-        pMachineInfo->setProcessType(process.getType());
-        StringBuffer displayName;
-        getProcessDisplayName(process.getType(), displayName);
-        pMachineInfo->setDisplayType(displayName.str());
+        if (isEclAgentProcess)
+        {
+            pMachineInfo->setProcessType(eqAgentExec);
```

Comment:
This looks a bit odd - if I am reading it correctly, it's checking if the process name is eclagent, and if so, setting the name and type to be agentexec instead (presumably to be later deduped). But would it not be simpler (and more correct) to skip the eclagent processes that are found? In particular, you don't want the fact that a (possibly orphaned) eclagent process is running to indicate to the user that agentexec is up, when it may not be,